docs/dev/tutorials/custom-aa-tutorial.md

```diff
@@ -6,6 +6,11 @@ In this tutorial, we build a factory that deploys 2-of-2 multisig accounts.
 <TocHeader />
 <TOC class="table-of-contents" :include-level="[2,3]" />
 
+::: warning
+
+Please note, that in the new `0.13.0` SDK version, the API/Node layer operates with gas. The ergs concept is used by VM only.
```

Comment:
I guess it is fine for the transaction period so that those familiar with the previous version of zkSync knew what is going on. 

@AnastasiiaVashchuk please make sure to use `0.13.1` version as `0.13.0` is outdated